lang/main-de.json

```diff
@@ -324,6 +324,13 @@
     "documentSharing": {
         "title": "Freigegebenes Dokument"
     },
+    "dialDtmf": {
```

Comment:
Not sure if we're supposed to add individual translations like this. Don't we usually have someone that updates the files in bulk, @saghul ?

Reply:
This is fine. We still accept translations as PRs, once we move to some system for translations, we will accept in PRs only main.json changes.